spec/imsc-hrm.html

```diff
@@ -242,6 +254,7 @@ <h2>Terms and Definitions</h2>
     <p>
       Unless noted otherwise, this specification applies to an <a>IMSC Document Instance</a>.
     </p>
+    <div class="issue" data-number="63"></div>
```

Comment:
I don't think citing this issue in the Conformance section is particularly helpful - I'd remove it from this one location.